### PR TITLE
Add methods to handle refunds

### PR DESCRIPTION
**Pull request by juanfra**
https://central.tri.be/issues/102081

Add methods to handle refunds for tickets and fix the attendees report accordingly.

### CODE REVIEW

File: src/Tribe/Ticket_Object.php [hunk 1]
```diff
@@ -166,6 +166,14 @@ class Tribe__Tickets__Ticket_Object {
 		 */
 		protected $qty_cancelled = 0;
 
+		/**
+		 * Number of tickets for which an order has been refunded.
+		 * Use $this->$qty_refunded( value ) to set manage and get the value
+		 *
+		 * @var int
```
Comment (jesseeproductions) [inline src/Tribe/Ticket_Object.php]: add @since TBD

File: src/Tribe/Ticket_Object.php [hunk 2]
```diff
@@ -866,6 +880,26 @@ public function qty_cancelled(  $value = null ) {
 			return $this->qty_cancelled;
 		}
 
+		/**
+		 * Method to manage the protected `qty_refunded` property of the Object
+		 * Prevents setting `qty_refunded` lower then zero
+		 *
+		 * @param int|null $value This will overwrite the old value
```
Comment (jesseeproductions) [inline src/Tribe/Ticket_Object.php]: add @since TBD

**Comment by jesseeproductions** [comment]
Some quick fixes then ready to merge.

**Comment by juanfra** [comment]
@jesseeproductions I still need approval with these new changes. Would you please take a look?

Thanks 😄

**Comment by juanfra** [comment]
Thanks so much, sir!

**Pull request merged**